source/source.go

```diff
@@ -48,62 +52,30 @@ func NewSource() sdk.Source {
 }
 
 // Parameters returns a map of named Parameters that describe how to configure the Source.
-func (s *Source) Parameters() map[string]sdk.Parameter {
-	return map[string]sdk.Parameter{
-		config.DSN: {
-			Default: "",
-			Validations: []sdk.Validation{
-				sdk.ValidationRequired{},
-			},
-			Description: "Data source name to connect to the Amazon Redshift.",
-		},
-		config.Table: {
-			Default: "",
-			Validations: []sdk.Validation{
-				sdk.ValidationRequired{},
-			},
-			Description: "Name of a table, the connector must read from.",
-		},
-		config.OrderingColumn: {
-			Default: "",
-			Validations: []sdk.Validation{
-				sdk.ValidationRequired{},
-			},
-			Description: "Column name that the connector will use for ordering rows. Column must contain unique " +
-				"values and suitable for sorting, otherwise the snapshot won't work correctly.",
-		},
-		config.Snapshot: {
-			Default:     "true",
-			Description: "Whether the connector will take a snapshot of the entire table before starting cdc mode.",
-		},
-		config.KeyColumns: {
-			Default:     "",
-			Description: "Comma-separated list of column names to build the sdk.Record.Key.",
-		},
-		config.BatchSize: {
-			Default:     "1000",
-			Description: "Size of rows batch. Min is 1 and max is 100000. The default is 1000.",
-		},
-	}
+func (s *Source) Parameters() commonsConfig.Parameters {
+	return s.config.Parameters()
 }
 
 // Configure parses and stores configurations,
 // returns an error in case of invalid configuration.
-func (s *Source) Configure(ctx context.Context, cfgRaw map[string]string) error {
+func (s *Source) Configure(ctx context.Context, cfgRaw commonsConfig.Config) error {
```

Comment:
why are we using common config here? why not the source config?

Reply:
It is an alias to "github.com/conduitio/conduit-commons/config".

Reply:
I know, I mean this `Configure` method is to configure the source configs, source configs include the common config with some extra params, which should be used here

Reply:
Sorry, I still didn't get it. Source struct implements Configure() and it has source/config.Config{} as one of its fields. And ParseConfig() parses the commonsConfig and copies it into source/config.Config{}.
Can you give me example/code-suggestion here? I believe I am missing something fundamental here.

Reply:
oh I was missing something here 😅 this commonConfig thing is newly added to conduit so I misunderstood it with something else! sorry for that!

Reply:
No worries!!